src/c++/perf_analyzer/inference_profiler.cc

```diff
@@ -866,6 +866,18 @@ InferenceProfiler::ValidLatencyMeasurement(
     }
   }
 
+  const double minimum_valid_requests_percent = 0.5;
```

Comment:
I want to better understand the reason we are dropping requests. I would expect this threshold to be something like 90%, but it sounds like even good cases don't hit this.

Reply:
There is a separate Jira issue for investigating/fixing the issue mentioned in this conversation. Can this conversation be resolved?

Reply:
Will this warning show up ever at 50%? Does it even show up in the really bad example cases that I gave you?

Reply:
I was also curious to learn more about how 50% has been obtained. For example, if a model responds to 1 request every 0.8 second, and we use a measurement interval of 2 seconds, more than 50% of the requests are valid and the throughput reported by PA will 1 infer/sec instead of 0.8 infer/sec (which is larger than the stability-percentage of 10%).